stix_shifter_modules/cybereason/README.md

```diff
@@ -0,0 +1,1323 @@
+                            "privileges": "UserPrivAdmin",
+                            "has_power_tool": "false",
+                            "total_machines": "1",
+                            "is_admin": "true",
+                            "unusual_process_ext_connection": "false",
+                            "is_local_system": "false",
+                            "organizational_unit": "INTEGRATION",
+                            "group_id": "513",
+                            "domain": "exlab1",
+                            "email_address": "Administrator@exlab1.local",
+                            "last_logged_machine": "siemplify-cyber"
+                        }
+                    },
+                    "user_id": "administrator",
+                    "account_created": "2021-04-08T14:22:08.000Z"
+                },
+                "1": {
+                    "type": "x-cybereason-malops",
+                    "has_suspicions": "false",
+                    "has_suspicious_process": "true",
+                    "has_malicious_process": "false"
+                },
+                "2": {
+                    "type": "email-addr",
+                    "value": "Administrator@exlab1.local",
+                    "belongs_to_ref": "0"
+                },
+                "3": {
+                    "type": "x-oca-asset",
+                    "host_name": "exlab1.local/Users/Administrator",
+                    "user_ref": "0"
+                }
+            },
+            "first_observed": "2021-04-08T14:22:08.000Z",
+            "last_observed": "2021-04-08T14:22:08.000Z",
+            "number_observed": 1
+        }
+    ],
+    "spec_version": "2.0"
+}
+
+```
+
+#### STIX Ping query
+```shell
+transmit
+cybereason
+"{\"host\":\"xx.xx.xx\",\"port\": xxxx}"
+"{\"auth\":{\"username\": \"xxxxx\", \"password\": \"xxxx\"}}"
+ping
+```
+
+#### STIX Ping query - output
+```json
+{
+    "success": true
+}
+```
+
+### Limitations
+- Cybereason does not support “OR” operator between the elements and features. It supports only "AND" operator through "connectionFeature" and "Filters".
+For more details on connectionFeature kindly follow the link : [Use multiple elements using Connection Feature](https://nest.cybereason.com/documentation/api-documentation/all-versions/use-multiple-elements-query "Use multiple elements using Connection Feature")
```

Comment:
these `nest.` links are not accessible. you can reference a public link or at least mention how to access.

Reply:
This link has been removed from  readme